src/main/java/gr/cytech/chatreminderbot/rest/GoogleCards/CardResponseBuilder.java

```diff
@@ -13,6 +14,9 @@
  */
 public class CardResponseBuilder {
```

Comment:
I'm not too happy with this class, we could certainly use some more type-safety by using POJO classes (same thing as for the 'request') which would also dramatically reduce the complexity